### PR TITLE
(DOCSP-7892): added php snippet for x509 conn

### PR DESCRIPTION
**Pull request by mongomoe**

### CODE REVIEW

File: source/includes/connection-snippets/x509-snippets/php-connection.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+/* Begin x509 connection */
+$certificateKeyFilePath = '/etc/certs/mongodb/client.pem';
+$client = new MongoDB\Client(
+    'mongodb+srv://<cluster-url>/test?authSource=$external&tlsCertificateKeyFile=' . $certificateKeyFilePath . '&retryWrites=true&w=majority&authMechanism=MONGODB-X509'
+);
+$collection = $client->testDB->testCol;
+$docCount = $collection->count([]);
+print($docCount);
```
Comment (jmikola) [inline source/includes/connection-snippets/x509-snippets/php-connection.php]: Same feedback as: https://github.com/mongodb/docs-ecosystem/pull/591#discussion_r352741661

File: source/includes/connection-snippets/x509-snippets/php-connection.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+/* Begin x509 connection */
+$certificateKeyFilePath = '/etc/certs/mongodb/client.pem';
+$client = new MongoDB\Client(
+    'mongodb+srv://<cluster-url>/test?authSource=$external&tlsCertificateKeyFile=' . $certificateKeyFilePath . '&retryWrites=true&w=majority&authMechanism=MONGODB-X509'
+);
+$collection = $client->testDB->testCol;
+$docCount = $collection->count([]);
```
Comment (jmikola) [inline source/includes/connection-snippets/x509-snippets/php-connection.php]: Same feedback as: https://github.com/mongodb/docs-ecosystem/pull/591#discussion_r352740158

File: source/includes/connection-snippets/x509-snippets/php-connection.php (hunk 1 above)
Comment (mongomoe) [reply source/includes/connection-snippets/x509-snippets/php-connection.php]: Cool fixed:
* made function call into `$collection->countDocuments()`

File: source/includes/connection-snippets/x509-snippets/php-connection.php (hunk 2 above)
Comment (mongomoe) [reply source/includes/connection-snippets/x509-snippets/php-connection.php]: Fixed:
* switched to `echo`

**Pull request merged**